src/cuga/backend/cuga_graph/utils/agent_loop.py

```
@@ -678,26 +719,88 @@ def get_output(self, event):
+        def _on_spawn_event(name: str, data: dict) -> None:
+            unified_queue.put_nowait((_SPAWN_TAG, name, data))
+
+        if agent_spawn_enabled:
+            _spawn_runtime.set_event_callback(_on_spawn_event)
+
+        async def _feed_graph():
+            exc_to_raise = None
+            try:
+                async for graph_event in self.get_stream(state, resume):
+                    await unified_queue.put((_GRAPH_TAG, graph_event))
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)